chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/UserPermissionsManagerImpl.java

```
@@ -120,4 +187,23 @@ public boolean evaluate(Object object) {
         return dtos.size() > 0;
     }
 
+    private Boolean doesUserTestingLabMapExist(Long atlId, Long userId) {
+        List<UserTestingLabMapDTO> dtos = userTestingLabMapDAO.getByUserId(userId);
+
+        if (dtos == null || dtos.size() == 0) {
+            // TODO: throw an exception throw exception...
+            LOGGER.error("Could not locate the UserTestingLabyMap object for Userid: " + userId + ", ATL: " + atlId);
```

Do you need to throw an exception?